### ISSUE TITLE
getSymbols with Yahoo not working

### ISSUE THREAD

**Issue by HLatte** [joshuaulrich/quantmod#182]
### Description
Error when trying to use getSymbols() with source = Yahoo. On the other hand, it works fine when using Google as a source.
Is Yahoo messing around with their servers again?

### Minimal, reproducible example

```r
# NOT working
require(quantmod)
getSymbols("YHOO", src = "yahoo")  
```
Retunrs:
> Error in download.file(paste(yahoo.URL, "s=", Symbols.name, "&a=", from.m,  : 
>   cannot open URL 'https://ichart.finance.yahoo.com/table.csv?s=YHOO&a=0&b=01&c=2007&d=6&e=29&f=2017&g=d&q=q&y=0&z=YHOO&x=.csv'
> In addition: Warning message:
> In download.file(paste(yahoo.URL, "s=", Symbols.name, "&a=", from.m,  :
>   InternetOpenUrl failed: 'The server name or address could not be resolved'

```r
# working
require(quantmod)
getSymbols("GOOG", src = "google") 
```
returns:
> [1] "GOOG"

### Session Info

```r
R version 3.4.0 (2017-04-21)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252  LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252 LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] quantmod_0.4-8       TTR_0.23-1           xts_0.9-7            zoo_1.8-0           
[5] RSQLite_1.1-2        RevoUtilsMath_10.0.0

loaded via a namespace (and not attached):
 [1] compiler_3.4.0   RevoUtils_10.0.4 DBI_0.6-1        tools_3.4.0      curl_2.6        
 [6] memoise_1.1.0    Rcpp_0.12.10     grid_3.4.0       digest_0.6.12    lattice_0.20-35 
```

**Comment by joshuaulrich** [comment]
Closing as duplicate of #157. This has been fixed for almost a month.  You're using 0.4-8 and this was fixed in 0.4-9.

**Comment by selgamal** [comment]
@HLatte ....in addition YHOO ticker does not exist anymore, most of yahoo operations was purchased by another company called Altaba Inc. (AABA), YHOO so stock doesn't exist anymore as far as I can tell
